app/jinja_filters.py

```diff
@@ -352,7 +352,8 @@ def format_unordered_list(context, list_items):
 
 @blueprint.app_template_filter()
 def concatenated_list(list_items, delimiter=', '):
-    return Markup(delimiter).join(item.strip() for item in list_items if item)
+    if list_items:
+        return Markup(delimiter).join(item.strip() for item in list_items if item)
```

Comment:
So, with the `format_address_list` jinja filter we get an error when all of the fields are empty strings. So if we have an empty list in this `concatenated_list` function it will error, I was just having a play about and accidentally added it, I can remove this and change format_address_list or add handling here? But it should be there in case there's an error

Reply:
i think `concatenated_list` is used by many other filters and there you no tests to support this.